src/QueryBuilder/HasPriceBetweenQueryBuilder.php

```diff
@@ -94,4 +97,20 @@ private function convertFromString(string $price): int
 
         return $transformer->reverseTransform($price);
     }
+
+    private function getDataByKey(array $data, ?string $key = null): ?string
+    {
+        return $data[$key] ?? null;
+    }
+
+    private function getQueryParamValue(?int $min, ?int $max): ?array
+    {
+        foreach (['gte' => $min, 'lte' => $max] as $key => $value) {
+            if (null !== $value) {
+                $paramValue[$key] = $value;
+            }
+        }
+
+        return $paramValue ?? null;
+    }
 }
```

Comment:
Missing new line

Reply:
it is but on GH is not visible :)
(you can check when you switch on branch)